nav/stepper/reference.py

```diff
@@ -13,7 +13,7 @@
 for pin in out_pins:
 	GPIO.setup(pin, GPIO.OUT)
 
-step_count = STEPS_PER_REVOLUTION * 5
+step_count = QUARTER_STEPS_PER_REVOLUTION * 5
```

Comment:
There should be some explanation about what QUARTER_STEPS_PER_REVOLUTION here. Moreover, this should be changed based on input from the user. Maybe it should be an instance of the class? @BenjiLee0002